.github/workflows/c-i.yaml

```diff
@@ -8,13 +8,16 @@ on:
     branches:
       - main
 
-permissions:
-  contents: read
-
 jobs:
   test:
+    strategy:
+      fail-fast: false
+      matrix:
+        runner:
+          - ubuntu-24.04
+          - ubuntu-24.04-arm
     name: Test Projects
-    runs-on: ubuntu-latest
+    runs-on: ${{matrix.runner}}
```

Comment:
is this really the only way to run on different arch?

and I can't even just do `runs-on: ubuntu-24.04-${{ matrix.arch }}` because github never registered ubuntu-24.04-x86 as an expllicit x86 ?